frontend/src/components/@common/Toast/ToastList.tsx

```diff
@@ -0,0 +1,15 @@
+import { styled } from 'styled-components';
+import type { PropsWithChildren } from 'react';
+
+const ToastList = ({ children }: PropsWithChildren) => {
+  return <Position>{children}</Position>;
+};
+
+export default ToastList;
+
+const Position = styled.div`
+  position: fixed;
+  bottom: 10%;
+  left: 50%;
+  transform: translate(-50%, 0);
+`;
```

Comment:
```suggestion
const ToastList = styled.div`
  position: fixed;
  bottom: 10%;
  left: 50%;
  transform: translate(-50%, 0);
`;

export default ToastList
```
1. 💬  위와 아래에 차이가 있는 것일까요? 스타일만 있는 컴포넌트인 것 같은데 따로 react component로 선언한 이유가 궁금해요. 
2. ToastList라고 하면 여러 Toast가 띄워지는 ui로 생각이 드는데, 나중에 구현할 것을 생각하고 이름을 지으신 걸까요?

Reply:
1. 개인적으로는 위쪽을 선호합니다~ 추후 컨벤션에 따라 styled.tsx 파일로 분리될 로직이라 따로 이야기 나눠보죠!
또 `ts vs tsx` 도 한번 이야기 나눠보고 싶네요!

2. 맞습니다~! 코난 리뷰에 비슷한 답변을 해서 링크 남길게요!
https://github.com/woowacourse-teams/2023-shook/pull/73#discussion_r1266821035